### PR TITLE
updated bacalahau rust code to work with compose CID structure. added…

### PR DESCRIPTION
**Pull request by stan36**
pull request to merge updated bacalahau

### CODE REVIEW

File: baclahauC/jsons/jsonCIDs.txt [hunk 1]
```diff
@@ -1,25 +1,18 @@
 test1.json : QmSCMeGh17KJWA9VHPftWqA1xdWM7McVtt3azxH3aenp1X
 test2.json : QmZhCRWWv8x6zJKhUKwnDNzerWoUaAjyeD4aSY8tGz6qE4
 test3.json : QmT3sHvzDBV53WqY2Z2Gt5U9zFPZYHKHMSKghbbkxt3zsM
+claim.jsom : QmTFcE6qwbpN5YozkR7u8yztZGYYjAztuxeF3VQMpAJwh1
```
Comment (ArnobChowdhury) [inline baclahauC/jsons/jsonCIDs.txt:4]: I think here is a typo. It should be `json`, right? Instead of `jsom`

File: baclahauC/jsons/claim.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "amount": 25,
+      "subject": "0x1413862C2B7054CDbfdc181B83962CB0FC11fD92",
+      "amountUnits": "ETH",
+      "rootClaimId": "kjzl6kcym7w8y5u49e724s7wh1j7zxch9ikwcy5klwe9tfhet8t894cqkvnjw3p",
+      "effectiveDate": 1675433754,
+      "claimSatisfactionStatus": "unsatisfied",
+      "id": "kjzl6kcym7w8ya7jhwkscslp9bopteordsemzu9uuq8b78ys11c00rie41hsg86"
+    }
+  ]
+}
```
Comment (ArnobChowdhury) [inline baclahauC/jsons/claim.json:35]: Do we need this hardcoded? Is this for testing?

**Review by ArnobChowdhury — APPROVED** [review]
Just couple of questions. If you think these are issues, just fix and merge.

File: baclahauC/jsons/jsonCIDs.txt (hunk 1 above)
Comment (stan36) [reply baclahauC/jsons/jsonCIDs.txt:4]: yea it should, good catch

File: baclahauC/jsons/claim.json (hunk 2 above)
Comment (stan36) [reply baclahauC/jsons/claim.json:35]: this is just for testing

**Pull request merged**